src/app/components/Timestamp/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,65 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Timestamp should render correctly 1`] = `
+.c0 {
+  font-size: 0.875em;
+  line-height: 1rem;
+  color: #5A5A5A;
+  font-family: ReithSansNewsRegular,Helvetica,Arial,sans-serif;
+  padding-bottom: 1rem;
+}
+
+@media (min-width:20em) and (max-width:37.4375em) {
+  .c0 {
+    line-height: 1.125rem;
+  }
+}
+
+@media (min-width:37.5em) {
+  .c0 {
+    font-size: 0.8125em;
+  }
+}
+
+<span
+  className="c0"
+>
+  <time
+    dateTime={1539969006}
+  >
+    October 19, 2018
```

Comment:
It's interesting that this is shown in the snapshot, whereas in several different browsers, I see the format 19 October 2018. 

Firefox nightly v64.0a1: 
<img width="117" alt="screen shot 2018-10-23 at 14 11 16" src="https://user-images.githubusercontent.com/3028997/47362924-7f77d680-d6cd-11e8-8b2c-713723f28519.png">

Chrome canary 72.0.3589:
<img width="111" alt="screen shot 2018-10-23 at 14 13 35" src="https://user-images.githubusercontent.com/3028997/47363066-d1206100-d6cd-11e8-8af8-fc4c0ba91a63.png">

Safari  11.1.2 : 
<img width="105" alt="screen shot 2018-10-23 at 14 11 51" src="https://user-images.githubusercontent.com/3028997/47363044-c4037200-d6cd-11e8-8e83-e5c3670dee09.png">

Chrome 38 via browserstack:
<img width="107" alt="screen shot 2018-10-23 at 14 14 42" src="https://user-images.githubusercontent.com/3028997/47363148-f8772e00-d6cd-11e8-8fb8-faae735d8288.png">
Given this date implementation is a temporary one, and will be replaced by using a more fully functional library in a future bit of work, I believe it should be okay to keep this implementation, even if the snapshot is incorrect.

Reply:
My guess is the page source will contain the comma but the browser rendering engines are ignoring the comma. But as you said this is a temporary solution.